src/azul/queues.py

```diff
@@ -386,17 +386,25 @@ def update_():
             time.sleep(3)
             mapping = lambda_.get_event_source_mapping(UUID=mapping['UUID'])
 
-    def manage_lambdas(self, queues: Mapping[str, Queue], enable: bool):
+    def functions_by_queue(self) -> Mapping[str, str]:
         """
-        Enable or disable the readers and writers of the given queues
+        Returns a mapping of queue names to the name of the Lambda function
+        triggered by the queue
```

Comment:
```suggestion
        triggered by the queue.
```

And this should say if the returned names are qualified or not.